security.md

```diff
@@ -0,0 +1,39 @@
+# Security
+
+Corteza takes the security of our software products and services seriously,
+which includes all source code repositories managed through our GitHub organizations.
+Here is the list of [Corteza Repositories](https://github.com/orgs/cortezaproject/repositories)
+
+If you believe you have found a security vulnerability in any Corteza repository, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please send email to [security@cortezaproject.org](mailto:security@cortezaproject.org).
+
+If you prefer to submit via forum, please report them to [Corteza Forum](https://forum.cortezaproject.org/).
```

Comment:
I would exclude the forum as it is public also, plus I think it's easier for us to track it if its via email (for now)

Reply:
Yeah sure.

Reply:
@vicpatel we'll have to use info@cortezaproject.org here instead of security@